packages/react-celo/src/utils/metamask.ts

```diff
@@ -57,6 +57,8 @@ export interface AddERC20TokenParameter {
   options: ERC20Token;
 }
 
+// It seems weird that this is defined in the metamask.ts file
+// I think there is something similar in the celo/base package. or celo/connect
```

Comment:
Guilty here, I couldn't find the interface I wanted anywhere in the monorepo, must have missed it